test/e2e/apimachinery/aggregator.go

```diff
@@ -26,6 +26,8 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/client-go/tools/cache"
+
```

Comment:
style: Should be in the same chunk of imports as the line below (remove the newline on 30)

Reply:
Fixed